apps/modernization-ui/src/design-system/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,36 @@
+import { Pagination as PaginationUswds } from '@trussworks/react-uswds';
+import styles from './pagination.module.scss';
+import { RangeToggle } from 'components/Table/RangeToggle/RangeToggle';
+import { useEffect } from 'react';
+import { usePage } from 'page';
+
+type Props = {
+    total: number;
+    pageSize: number;
+};
+
+export const Pagination = ({ total, pageSize }: Props) => {
+    const { ready, page } = usePage();
+
+    useEffect(() => {
+        ready(total ?? 0, page.current);
+    }, [total, page]);
+
+    return (
+        <div className={styles.pagination}>
+            <div className={styles.range}>
+                Showing <RangeToggle /> of <span id="totalRowCount">{total}</span>
```

Comment:
issue:  the `initial` value of the `RangeToggle` needs to be set to the `pageSize` from the PageState or else it reverts back to the default value of `10` when choosing another size.